src/Plugin/SapiHandlerAnnotationManager.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\sapi\Plugin\SapiHandlerAnnotationManager.
+ */
+
+namespace Drupal\sapi\Plugin;
+
+use Drupal\Core\Plugin\DefaultPluginManager;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+
+/**
+ * Provides the Sapi handler annotation plugin manager.
+ */
+class SapiHandlerAnnotationManager extends DefaultPluginManager {
```

Comment:
Assuming that our plugins are called (annotated as) `StatisticsPlugin`, `StatisticsPluginManager` would make a fine, fine name.